README.md

```diff
@@ -115,6 +119,8 @@ Returns the `trap`.
 These options are used to override the focus trap's default behavior for this particular activation.
 
 - **onActivate** `{() => void | null | false}`: Default: whatever you chose for `createOptions.onActivate`. `null` or `false` are the equivalent of a `noop`.
```

Comment:
Not sure why we have `void | null | false` here. It can just be `void` since we ignore the return value, whatever it is, and no need for "`null` or `false` are the equivalent of a `noop`." at the end.